config.json.example

```diff
@@ -113,5 +113,6 @@
       "CLOUDKARAFKA_PASSWORD": "",
       "CLOUDKARAFKA_TOPIC_PREFIX": ""
     },
-    "STACK_IMPACT_KEY": "aaaabbbbccccddddeeeeffffgggg111100002222"
+    "STACK_IMPACT_KEY": "aaaabbbbccccddddeeeeffffgggg111100002222",
+    "CHAT_APPROVAL_REQUIRED": "false"
```

Comment:
Is there any way we could turn this feature on and off using a setting in the Hall? That would allow moderators to manage it fully. If that's possible but difficult, it wouldn't need to be done in this PR.

I guess it would pretty much need to be a setting in the database, which would complicate things. A new collection? A value in the Tavern's document? I suppose we don't really have support for this sort of thing at the moment.

Reply:
Yea, we don't have any good support for this.